CONTRIBUTING.adoc

```diff
@@ -612,25 +612,82 @@ vector: '/images/logos/${imageId}/${imageId}.svg'
+
+Both teams operate under the umbrella of link:https://www.jenkins.io/sigs/docs/[Jenkins Documentation Special Interest Group] led by the link:https://www.jenkins.io/project/team-leads/#documentation[Documentation Officer].
+
+=== Joining the teams
+
+If you are interested to join the Triage or Copy Editors team, 
+you can request membership in the link:https://groups.google.com/d/forum/jenkinsci-dev[Jenkins Developer mailing list] or in the link:https://www.jenkins.io/sigs/docs/[Documentation SIG channels].
+The request will be processed and discussed by the community, and then the link:https://www.jenkins.io/project/team-leads/#documentation[documentation officer] will make a decision.
+
+Eligibility requirements:
+
+* Membership in both teams requires a track of contributions to the Jenkins website and/or documentation.
+  _Triage_ team is effectively an onboarding team for contributors interested in becoming copy editors,
+  and this team has a low entry bar.
+* Applicants to the _Copy Editors_ team should have a signed link:https://github.com/jenkinsci/infra-cla[Contributor License Agreement].
 
 [[reviewing]]
-== Reviewing changes
+=== Reviewing changes
 
 There are many pull requests being submitted to jenkins.io every week.
 Reviews are driven by the community, and any contributions are always welcome.
-There is also a link:https://github.com/orgs/jenkins-infra/teams/copy-editors[Copy Editors] team which performs reviews and merges, 
-but such reviews may take some time depending on availability of contributors.
+Reviews may take some time depending on availability of contributors.
 
-Some tips:
+Some tips for contributors:
 
 * Pull requests are open to public, and any GitHub user can review changes and provide feedback.
   If you are interested to review changes, please just do so (and thanks in advance!). 
   No special permissions needed
 * If you need help with reviews for documentation changes,
-  you can in the link:https://gitter.im/jenkinsci/docs[Documentation SIG Gitter channel].
-* If you are interested to join the Copy Editors team, 
-  you can request membership in the link:https://groups.google.com/d/forum/jenkinsci-dev[Jenkins Developer mailing list].
-  Such membership requires a track of contributions to Jenkins, and also a signed link:https://github.com/jenkinsci/infra-cla[Contributor License Agreement]
+  you can ask in the link:https://gitter.im/jenkinsci/docs[Documentation SIG Gitter channel].
+
+[[merging-common]]
+=== Merging changes in common areas
+
+Common area process applies when there is no special ownership or process defined.
+Pull requests to common areas can be merged by any _Copy Editor_ once...
+
+* Conversations in the pull request are over OR it is explicit that a reviewer does not block the change (often indicated by line comments attached to an approving PR review, or by using the term "nit", from "nit-picking")
```

Comment:
Can we ”Conversation in the pull request are completed”

Reply:
what's wrong with the current text from your point of view? It is how it is written in https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc

Reply:
It's now the word ”over”. It feels like it should be ”completed”. Over on sound proper

Reply:
```suggestion
* Conversations in the pull request are completed OR it is explicit that a reviewer does not block the change (often indicated by line comments attached to an approving PR review, or by using the term "nit", from "nit-picking")
```